src/api/posts/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@senecacdot/posts-service",
+  "version": "1.0.0",
+  "private": true,
+  "description": "A service for retrieving posts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
We should add a couple of scripts so you can run this locally (start, dev).  See image service.